apps/examples/src/styles.css

```diff
@@ -88,12 +86,65 @@ html,
 	}
 }
 
+.accordion__trigger{
```

Comment:
nit: (does prettier take care of this?) should be a space before `{` (and below)

Reply:
Weirdly, it doesn't. Might have to ask someone about this.